padrino/manifest.yml

```diff
@@ -0,0 +1,3 @@
+---
+applications:
+-  command: bundle exec padrino start -p $PORT
```

Comment:
Also, are your sure `$PORT` is matching the CF-generated port? Maybe use `$VCAP_APP_PORT` instead?